app/helpers/application_helper.rb

```diff
@@ -1,7 +1,7 @@
 module ApplicationHelper
   def display_picture(game)
```

Comment:
Need to remove the space before 'def' to pass the test. Probably need to re-indent all the following code.